### PR TITLE
EI S01-S04x: Add more po hints

### PR DESCRIPTION
**Pull request by stevecotton**
No changes to translatable strings, just to the hints.

@tamanegi-jp and @hermestrismi, is this the right sort and amount of comments? I'm intending to do another pass over the same scenario files and maybe add a bit more, hence the "draft" status of the PR.

### CODE REVIEW

**Comment by hermestrismi** [comment]
In my opinion this is more than what I hoped for. Fine for me. Good work 👏

**Comment by tamanegi-jp** [comment]
Me too. Good quality and editor (poedit, transifex etc.) friendly tips are of course welcome.

**Review by Dalas121 — APPROVED** [review]
These all look correct to me; thanks for taking the time to go through and improve things.

`wmllint: local spelling warlordship`
^ unused; the old EI included "warlordship" in that message, I must have neglected to remove the wmllint comment.

**Comment by stevecotton** [comment]
Thanks for the feedback. The force-push was just to remove the .po files again, it's back to exactly the version that previously passed the CI (but I forgot about the "warlordship" comment").

**Comment by stevecotton** [comment]
These are the only changes in the force push:
* comments for S03, which didn't have any added in the previous versions of the PR
* remove the thing about warlordship
* in S04a, add "#po: the mercenaries decide it's time to retreat"

File: data/campaigns/Eastern_Invasion/scenarios/01_Eastern_Invasion.cfg [hunk 1]
```diff
@@ -51,9 +51,9 @@
             {EI_BIGMAP}
         [/part]
         [part]
-            # The River Guard posts had been built in 470 YW. They were
-            # abandoned in 544 YW; the wave of colonization had begun around
-            # 530 YW.
+            #po: The River Guard posts had been built in 470 YW. They were
+            #po: abandoned in 544 YW; the wave of colonization had begun around
+            #po: 530 YW. It is now 625 YW.
```
Comment (stevecotton) [inline data/campaigns/Eastern_Invasion/scenarios/01_Eastern_Invasion.cfg:56]: This intro starts in 625 YW; the king's forces arrive at the outposts in 626 YW.

**Pull request merged**